pkg/export/export.go

```diff
@@ -400,16 +408,16 @@ func (e *Exporter) Export(metadata MetadataFunc, batch []record.RefSample) {
 		prometheusSamplesDiscarded.WithLabelValues("no-ha-range").Inc()
```

Comment:
shouldn't you be adding to `exemplarsDropped` here?

Reply:
yes missed a spot

Reply:
or actually it should be `discarded` here, unless this should also be `prometheusSamplesDropped`

Reply:
actually is this call even correct? It looks like it increments the `discarded` count once, but it looks like it throws out everything?

Reply:
Okay I see you fixed this in https://github.com/GoogleCloudPlatform/prometheus-engine/pull/366, so once that is merged into `main` I will pull those changes in and update here.